ssl-config/src/main/java/com/palantir/remoting/ssl/SslSocketFactories.java

```diff
@@ -75,23 +76,24 @@ private static TrustManagerFactory createTrustManagerFactory(TrustStoreConfigura
         }
     }
 
-    private static KeyManagerFactory createKeyManagerFactory(KeyStoreConfiguration keyStoreConfig) {
+    private static KeyManagerFactory createKeyManagerFactory(SslConfiguration sslConfig) {
```

Comment:
the args to this should probably explicitly be `Path keyStorePath`, `String keyStorePassword`, `Optional<String> keyStoreKeyAlias`